src/System.Diagnostics.EventLog/tests/EventLogTests/EventLogTests.cs

```diff
@@ -1,12 +1,10 @@
-﻿using System.Diagnostics;
-using Xunit;
-using System;
+﻿using Xunit;
 
 namespace System.Diagnostics.Tests
 {
     public class EventLogTests : EventLogTestsBase
     {
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
I should have suggested it before, but I believey ou can write as just
`[ConditionalFact(nameof(PlatformDetection.IsNotWindowsNanoServer))]`